src/components/Settings.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import styled from 'styled-components';
+import NavBar from '../components/NavBar';
+
+const SettingsContainer = styled.div`
+  color: palevioletred;
+`;
+
+const SettingsTitle = styled.h1``;
+
+const Settings = () => {
+  return (
+    <SettingsContainer>
+      <NavBar />
+      <SettingsTitle>Hi you're here to change some settings</SettingsTitle>
```

Comment:
It seems like something with the syntax here is wrong, maybe the single quotes makes it weird? Not sure how it looks like in atom.

Reply:
It looks fine on atom and it seems to render without errors :confused:

Reply:
I guess you could use the html symbol, but it's definitely not a biggie :woman_shrugging: